tests/acceptance/features/apiGraph/fullSearch.feature

```diff
@@ -89,3 +89,23 @@ Feature: full text search
       | old              |
       | new              |
       | spaces           |
+
+
+  Scenario Outline: search restored files through a tag
+    Given using <dav-path-version> DAV path
+    And user "Alice" has uploaded file with content "hello world" to "file1.txt"
+    And user "Alice" has uploaded file with content "Namaste nepal" to "file2.txt"
+    And user "Alice" has created the following tags for file "file1.txt" of the space "Personal":
+      | tag1 |
+    And user "Alice" has deleted file "/file1.txt"
+    And user "Alice" has restored the file with original path "/file1.txt"
+    When user "Alice" searches for "Tags:tag1" using the WebDAV API
+    Then the HTTP status code should be "207"
+    And the search result should contain "1" entries
+    And the search result of user "Alice" should contain these entries:
+      | /file1.txt |
```

Comment:
```suggestion
    And the search result of user "Alice" should contain only these files:
      | file1.txt |
```
Single step can be used instead of those two steps.

Reply:
ok